apis/extension/reservation.go

```diff
@@ -34,13 +34,38 @@ const (
 
 	// AnnotationReservationAllocated represents the reservation allocated by the pod.
 	AnnotationReservationAllocated = SchedulingDomainPrefix + "/reservation-allocated"
+
+	// AnnotationReservationAffinity represents the constraints of Pod selection Reservation
+	AnnotationReservationAffinity = SchedulingDomainPrefix + "/reservation-affinity"
```

Comment:
When the Pod declares the annotation, does the selection condition of the Reservation for the Pod still take effect?